### PR TITLE
Added language parameter to support i18n of Dark Sky API

### PR DESCRIPTION
**Pull request by nodomain**
Added the ability to control the language parameter for Dark Sky. This is needed for my modifications for the Home Assistant Dark Sky Sensor where I want to have german weather forecasts :-)
So it would be great if you could release a new version of your library and I'll raise the version of the dependency accordingly in order to finish my Home Assistant pull request.

Thanks!

### CODE REVIEW

**Comment by nodomain** [comment]
Ooops. Fixing the tests.

**Comment by nodomain** [comment]
I messed up with the API key for the tests. Would you please have a look? First time Travis user...

**Comment by riemers** [comment]
Would love to see support for this too, could close a 3+ years old ticket then too https://github.com/ZeevG/python-forecast.io/issues/25 🌘

**Comment by cgtobi** [comment]
Can you please merge this? Language support would be really great and the code changes are minimal.

**Comment by suhajdab** [comment]
+1

**Comment by jokerigno** [comment]
Any news on this update? Thank you!

File: example.py [hunk 1]
```diff
@@ -11,9 +11,10 @@ def main():
 
     lat = -31.967819
     lng = 115.87718
-    time = datetime.datetime(2015, 2, 27, 6, 0, 0)
+    lang = "de"
```
Comment (lolorc) [inline example.py]: why would the default change to "de" ? ;-)

**Comment by ZeevG** [comment]
Thanks for the PR and apologies it took me so long to look at it. I've now merged it and the changes have been released in 1.4.0

**Comment by cgtobi** [comment]
Great. Thanks ZeevG.

**Comment by jokerigno** [comment]
Great! Thank you!

**Pull request merged**